README.md

```diff
@@ -531,11 +535,21 @@ Statics parameters
 - `jsCompilerWhitelist`: Whitelist of JS files to compile as an array. Leave undefined to compile all files. Supply a `:` character after each file name to delimit an alternate file path and/or file name for the minified file.
   - Default: `null` (compiles all JS files, if a JS minifier is enabled).
   - Example: *[String]* `library-name/example.js:lib/example.min.js` (compiles `library-name/example.js` into `lib/example.min.js`).
+  - `jsCompilerBlacklist`: Array of JS files to exempt from minification. These files will be copied as is to the build folder. Leave undefined to compile all files.
```

Comment:
revise the whitelist language to match this new, better language: "Array of JS files to whitelist for minification."